hack/build.sh

```diff
@@ -92,12 +92,13 @@ for dir in ${dirs}; do
   fi
 
   if [[ -v TEST_MODE ]]; then
-    VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run
-
+    TEST_DIR=../hack
+    pushd $TEST_DIR && make $TEST_CASE VERSION=$dir IMAGE_NAME=${IMAGE_NAME}
```

Comment:
Actually, it is not really pretty calling the make here (but that is rather historic issue than problem with potential fix).  The `test/run` script should be run from `common.mk` somehow (if someone wanted to make that pretty).